array_api_compat/common/_helpers.py

```diff
@@ -219,6 +236,13 @@ def to_device(x: "Array", device: "Device", /, *, stream: "Optional[Union[int, A
         return _cupy_to_device(x, device, stream=stream)
     elif _is_torch_array(x):
         return _torch_to_device(x, device, stream=stream)
+    elif _is_dask_array(x):
+        if stream is not None:
+            raise ValueError("The stream argument to to_device() is not supported")
+        # TODO: What if our array is on the GPU already?
```

Comment:
I think dask supports cupy on the GPU.

Is this something we also need to take into consideration?

Reply:
Yes indeed, it does support CuPy. I"m not sure that a `stream` input can be supported though. If a Dask array spans multiple machines, I think there'd be multiple streams with numbers that are unrelated to each other. Which therefore can't be supported at the Dask level in this API.

That's probably fine - you'd only move arrays in a single process to another device like this I think, so maybe the whole `to_device` method doesn't quite work for Dask? @jakirkham any thoughts on this?

Reply:
Even if dask can't really support device transfers or anything I would not call the default device "cpu" as that's misleading. We could just create a proxy `DaskDevice` object that serves as the device for dask arrays.

Reply:
@lithomas1 what's the status of this comment? Does the `device()` helper above need to be updated for dask?